### ISSUE TITLE
Centered text on interior callouts

### ISSUE THREAD

**Issue by ScottKillen** [efemkay/obsidian-modular-css-layout#32]
After release 0.9.0, text is centered inside callouts nested within a multi-column callout.

Theme is **minimal**.

Code is:

```
> [!multi-column]
>
>> [!info]
>> Information!!
>
>
>> [!warning]
>> ## Why?
>> - This spacing is strange
>> - This list looks funny.
```

![image](https://user-images.githubusercontent.com/1509989/229293292-bf19137d-2c6b-404d-bb9b-187f7b59c9bc.png)

**Comment by efemkay** [comment]
@ScottKillen yeah I noticed that and just quitely update it again but didn't create a release. Just update snippet again. Let me know if the issue persists
